images/README.md

```diff
@@ -164,31 +165,32 @@ export DAOS_SOURCE_IMAGE_FAMILY="rocky-linux-8"
 Say you want to make the following customizations to the images:
 
 1. Change the image family names of the DAOS Server and DAOS Client images
-2. Use the `rocky-linux-8-optimized-gcp` source image for the DAOS server image.
-3. Use the `rocky-linux-8` source image for the DAOS client image.
+2. Use `hpc-rocky-linux-8` as the source image for the DAOS client image (the default).
+3. Use `rocky-linux-8-optimized-gcp` as the source image for the DAOS server image.
 
-In this scenario since you want to use a different source image for the server and client images, it will be necessary to run `build.sh` twice with different environment variables.
+In this scenario it will be necessary to run the `build.sh` script two times with
+different environment variables.
 
-**Build Server Image**
-
-No need to set the `DAOS_SOURCE_IMAGE_FAMILY` since we want the default value `rocky-linux-8-optimized-gcp` for the server image.
+**Build Client Image**
 
 ```bash
 cd images
-export DAOS_BUILD_CLIENT_IMAGE="false"            # Do not build client image
-export DAOS_SERVER_IMAGE_FAMILY="my-daos-server"  # Change image family name
+export DAOS_BUILD_CLIENT_IMAGE="true"
+export DAOS_CLIENT_IMAGE_FAMILY="daos-client"
+
+export DAOS_BUILD_SERVER_IMAGE="false"
```

Comment:
Need comment to be consistence with other line like # Do not build server image

Reply:
Fixed